hewr/R/to_epiweekly_quantile_table.R

```diff
@@ -1,3 +1,13 @@
+process_posterior_for_table <- function(file, last_training_date) {
+  arrow::read_parquet(file) |>
+    dplyr::filter(.data$date > last_training_date) |>
```

Comment:
For generality, maybe name the argument `first_target_date` and use a `>=`?  Sometimes (nowcasting problems) the first target date is before the last training date.

```suggestion
process_posterior_for_table <- function(file, first_target_date) {
  arrow::read_parquet(file) |>
    dplyr::filter(.data$date > first_target_date) |>
```

Reply:
https://github.com/CDCgov/pyrenew-hew/pull/287/commits/b3d348e940f99bdd303fe187bd6792d7ea2af9c6